taxi/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf import settings
+from django.conf.urls.static import static
+from django.urls import path
+
+from taxi.views import index
+
+urlpatterns = [
+    path("", index, name="index")
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
`+ static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)` should be in the `taxi_service/urls.py`